src/services/quest-data.js

```diff
@@ -0,0 +1,97 @@
+const pants = {
+    id: 'pants',
+    title: 'Pick a Bottom',
+    image: 'pantsquest.png',
+    description: `
```

Comment:
All of these back ticks `` should be single quotes. It will require a bit or reformatting things, feel free to as for help if you need it.

Reply:
It's because it's a multi-line string (IDK, another TA told me to say that, I wont say who, I have no problem changing it if needed.)